tldr-page.go

```diff
@@ -19,6 +19,13 @@ func removesuffix(input string) string { // Function used for removing trailing
 	temp := strings.TrimSpace(input)
 	return temp
 }
+func remove_punctuation(input string) string { //trims punctuation
+	temp := strings.TrimSuffix(input,":")
+	temp = strings.TrimSuffix(temp,".")
+	temp = strings.TrimSuffix(temp,",")
+	temp = strings.TrimSuffix(temp,"!")
```

Comment:
```suggestion
	temp := strings.TrimRight(input, "!.-\",:")
```

Reply:
I like this suggestion more since it also does the otherside and removes the need for the usage of removesuffix()

```suggestion
	temp := strings.TrimRight(input, "!.-\",:` ")
	temp = strings.TrimLeft(input, "-<>`# ")
```